src/templates/fundraising/components/ShareInfo.js

```diff
@@ -17,40 +18,49 @@ function ShareInfo({
         `}
       >
-        <Text.Paragraph>
-          The shareholders are the one contributing to the fundraising campaign.
-          They are represented through a custom bonded-token - they can buy and
-          redeem through the Aragon Fundraising interface - and a voting app.
-          They hold most of the rights over the organization.
-        </Text.Paragraph>
-        <Text.Paragraph>
-          <Text weight="bold">Handling system.</Text> Shareholders decide on
-          which apps are to be installed, which apps are to to upgraded and how
-          permissions are to be set.
-        </Text.Paragraph>
-        <Text.Paragraph>
-          <Text weight="bold">Handling fundraising parameters.</Text>{' '}
-          Shareholders decide on whether / how beneficiary, fees,
-          collateralization settings and collaterals taps should be updated.
-        </Text.Paragraph>
+        <Paragraph>
+          The shareholders are the ones contributing to the fundraising
+          campaign. They are represented through a custom bonded-token and a
+          voting app. They hold most of the governance rights over the
+          organization.
+        </Paragraph>
+        <Paragraph>Shareholders can:</Paragraph>
+        <Paragraph>
+          <strong>Buy and redeem tokens.</strong> Shareholders can buy and
+          redeem tokens through the Aragon Fundraising interface.
+        </Paragraph>
+        <Paragraph>
+          <strong>Handle fundraising parameters.</strong> Shareholders decide on
+          how beneficiary, fees, and collateralization settings should be
+          updated. They also control the amount of funds automatically
+          transferred to the board each month.
+        </Paragraph>
+        <Paragraph>
+          <strong>Handle organization settings.</strong> Shareholders decide on
+          which apps are installed or upgraded and which permissions are set.
+        </Paragraph>
       </Info>
       <Info
         css={`
           margin-bottom: ${3 * GU}px;
         `}
       >
-        <Text.Paragraph>
-          This architecture grants most of the governance rights to shareholders
-          [to protect their investment]. There is thus a need to mitigate
-          situations where a shareholder owning more than 50% of the shares
-          would own the whole organization. This is why shareholders votes [i.e.
-          most of the organization decisions] can only be open and initiated by
-          the board.
-        </Text.Paragraph>
+        <Paragraph>
+          This architecture grants most of the governance rights to
+          shareholders, to protect their investment. However, this also requires
+          the organization to be able to mitigate situations where a shareholder
+          could own the whole organization by owning more than 50% of the
+          shares.
+        </Paragraph>
+        <Paragraph>
+          This is why shareholder votes, where most of the organization's
```

Comment:
While we are at it :D

```suggestion
          This is why shareholder votes, where most of the organization’s
```